### DISCUSSION TITLE
Gotchas when changing postgres password?

### DISCUSSION THREAD

**Discussion by emschwartz** [supabase/supabase#2339] [Questions] [answered]
Are there any Supabase services that use the `postgres` database user? I'm wondering whether it's safe to change that user's password or if that will break something.

Thanks!

**Comment by dragarcia** [comment, marked as answer]
Hi @emschwartz 👋 

None of the services use the `postgres` DB user. You can go ahead with changing it!

**Reply by emschwartz** [reply]
Great, thanks!
